contents/product-engineers/pmf-surveys.md

```diff
@@ -0,0 +1,155 @@
+
+Ellis recommend a minimum of 30 responses before the survey becomes directionally useful, and that 100+ responses is ideal.
+
+### Drawbacks of PMF surveys
+
+Calculating your PMF score is not enough. The *real* insights come from segmenting your users based on their response, and then asking follow-up questions – either in a [survey](/surveys) or by [conducting interviews](/product-engineers/interview-snapshot-guide).
+
+Here are five follow-up questions you should ask:
+
+1. Why did you select this answer? (after responding to the main survey question)
+   
+2. What is the primary benefit that you have received from our product?
+
+3. What would you use as an alternative to our product?
+
+4. What type of person do you think would benefit most from FYI?
+
+5. How can we improve our product?
+
+Answers to these questions is often surprising – how your users view your product might not be the same as how you do.
+
+### Who should answer the PMF survey?
+
+Ellis recommends sending the PMF survey to users that have recently had meaningful interactions with your product. This means users that have:
+
+- experienced the core value-prop of your product.
+- used your product at least twice.
+- used your product in the last two weeks.
+
+This way, you ensure that the feedback you receive is relevant and accurate.
+
+## How to use PMF to improve your app
+
+Rahul Vohra, founder of [Superhuman](https://superhuman.com), famously used PMF surveys to build an [engine to find product-market fit](https://review.firstround.com/how-superhuman-built-an-engine-to-find-product-market-fit). This catapulted them to over 50,000 paying customers and whopping **$20 million** in revenue!
+
+At a high-level, here are the steps you need to follow to do the same for your app (we'll dive a [case study of Superhuman below](#case-study-superhuman)):
+
+1. Send the PMF survey to your users.
+
+2. Identify your best users i.e., those who answered "very disappointed".
+
+3. Paint a clear picture of who these people are by creating a [user persona](/product-engineers/how-to-create-user-personas) of them.
+
+4. Find out why they love your product. Do this by asking `"What is the main benefit you receive from our product"` as follow-up question in your survey.
+
+5. Next, direct your attention to the "somewhat disappointed" users. Focus only on those who match the user persona you created in `step 3` *and* responded the same to "main benefit" question in `step 4`.
+
+6. It's clear that your app resonates with them, but something is holding them back from *loving* it. Find out what's missing or annoying for them. Do this by asking `How can we improve our app for you?`. 
+
+7. Reprioritize your roadmap to address their issues and ship your changes.
+
+8. Repeat the process to convert your "somewhat disappointed" users into your best users.
+
+![How to improve your app with PMF survey](../images/blog/pmf-surveys/pmf-survey-steps.png)
```

Comment:
I'm a bit unsure of this image. I like it to some extent, but I think it may too hard to parse (perhaps because of too much text)